dora/shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -83,7 +84,8 @@ enum Command {
     SUMMARY, // Report cluster summary
     UFS, // Report under filesystem information
     JOBSERVICE, // Report job service metrics information
-    PROXY // Report proxy information in the cluster
+    PROXY, // Report proxy information in the cluster
```

Comment:
we use `bin/alluxio fsadmin report capacity` to list all workers and their capacity. Does that overlap with what you are doing here?

Reply:
this doesn't report any worker capacity info, only the live/fail status of each worker